tests/unit/DebugUtilsTest.ts

```diff
@@ -693,46 +693,6 @@ describe('DebugUtils', () => {
             });
             expect(reason).toBe('debug.reasonVisibleInLHN.pinnedByUser');
         });
-        it('returns correct reason when report has IOU violations', async () => {
```

Comment:
Deleted this since it seems to me it's testing the same thing as [this other test](https://github.com/Expensify/App/pull/51893/files#diff-d4ca88de0f9a19582a7bd9950a945f66dd78a2b624fc2ef3109c29891a4fbd5cL1533)

Reply:
Why have 2 tests that do the same? It's confusing, useless and a waste of time to have to update it now (and in the future whenever this changes)

Reply:
Cause I got the impression that we think they're identical, but not 100% sure. If that's the case, I think it's better to leave it, but NAB